### PR TITLE
Add CTemplar

### PR DESCRIPTION
**Pull request by dngray**
## Description

Resolves: https://github.com/privacyguides/privacyguides.org/discussions/131
Resolves: https://github.com/privacytools/privacytools.io/issues/1642

Edit: still need to fix logo, for light/dark version

### CODE REVIEW

File: _includes/legacy/sections/email-providers.html [hunk 1]
```diff
@@ -234,4 +234,41 @@ <h5>{% include badge.html color="info" text="Extra Functionality" %}</h5>
...
+    <p>CTemplar supports importing <a href="https://ctemplar.com/help/answer/importing-contacts/">contacts</a> and <a href="https://ctemplar.com/help/answer/contact-encryption/">contacts are encrypted</a> at rest however, they are only accessible in the webmail and apps.</p>
+
+    <h5>{% include badge.html color="success" text="Email Encryption" %}</h5>
+    <p>CTemplar has <a href="https://ctemplar.com/help/answer/how-does-encryption-decryption-work-in-ctemplar/">integrated encryption</a> in their webmail, which simplifies sending messages to users with public OpenPGP keys.</p>
+
+    <h5>{% include badge.html color="success" text=".onion Service" %}</h5>
+    <p>CTemplar's login and services are accessible over Tor, <a href="http://ctemplarpizuduxk3fkwrieizstx33kg5chlvrh37nz73pv5smsvl6ad.onion /">ctemplarpizuduxk3fkwrieizstx33kg5chlvrh37nz73pv5smsvl6ad.onion</a></p>
+
+    <h5>{% include badge.html color="info" text="Extra Functionality" %}</h5>
+    <p>TBD</p>
```
Comment (Mikaela) [inline _includes/legacy/sections/email-providers.html:271]: Should this be a draft PR?

**Review by Mikaela — CHANGES_REQUESTED** [review]
This TBD line doesn't seem like it belongs here.

File: _includes/legacy/sections/email-providers.html (hunk 1 above)
Comment (dngray) [reply _includes/legacy/sections/email-providers.html:271]: Yes it is.

**Comment by pm4rcin** [comment]
I have some problems with adding CTemplar.
1. They had no backup and data loss occurred. There was no e-mail or any information on the front page. I had only a few messages and didn't lose any but it was enough to close the account. I discovered that it happened accidentally by looking at privacytools Github. I wouldn't know about it otherwise. I really liked transparency like e.g. Gitlab did with their failure handling and this is completely opposite. Here's some info:
https://github.com/privacytools/privacytools.io/issues/1642#issuecomment-878255344
2. Regarding tor support they have disabled it today as reported on their twitter: [Ctemplar Twitter](https://nitter.net/RealCTemplar/status/1458775445202157570). I have checked it manually and when logging in it redirects to ctemplar.com.

**Comment by dngray** [comment]
> They had no backup and data loss occurred. There was no e-mail or any information on the front page. I had only a few messages and didn't lose any but it was enough to close the account. I discovered that it happened accidentally by looking at privacytools Github. I wouldn't know about it otherwise. I really liked transparency like e.g. Gitlab did with their failure handling and this is completely opposite. Here's some info:
> https://github.com/privacytools/privacytools.io/issues/1642#issuecomment-878255344

We're aware of this issue and while it isn't ideal, it does seem that the company has taken some serious steps to make sure it never happens again https://github.com/privacyguides/privacyguides.org/discussions/131#discussioncomment-1581612

> Regarding tor support they have disabled it today as reported on their twitter: [Ctemplar Twitter](https://nitter.net/RealCTemplar/status/1458775445202157570). I have checked it manually and when logging in it redirects to ctemplar.com.

That is unfortunate.

**Comment by OneWhiteBird** [comment]
[SVG.zip](https://github.com/privacyguides/privacyguides.org/files/7526684/SVG.zip)

Edited logos, as requested!

**Comment by dngray** [comment]
> Regarding tor support they have disabled it today as reported on their twitter: [Ctemplar Twitter](https://nitter.net/RealCTemplar/status/1458775445202157570). I have checked it manually and when logging in it redirects to ctemplar.com.

Still seems to be the case
```
> GET / HTTP/1.1
> Host: mail.ctemplarpizuduxk3fkwrieizstx33kg5chlvrh37nz73pv5smsvl6ad.onion
> User-Agent: curl/7.80.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 301 Moved Permanently
< Server: nginx
< Date: Wed, 01 Dec 2021 13:15:40 GMT
< Content-Type: text/html
< Content-Length: 178
< Connection: keep-alive
< Location: https://mail.ctemplar.com/
<
```

Worth noting that Protonmail still has this enabled
```
> GET /login HTTP/1.1
> Host: protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion
> User-Agent: curl/7.80.0
> Accept: */*
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< date: Wed, 01 Dec 2021 13:18:32 GMT
< last-modified: Sun, 31 Jan 2021 09:41:46 GMT
< accept-ranges: bytes
< content-length: 3309
< vary: Accept-Encoding
< cache-control: max-age=0, no-cache, no-store, must-revalidate
< expires: Wed, 11 Jan 1984 05:00:00 GMT
< set-cookie: Session-Id=Yad2KC3wGVsmJJPzMw5P0QAAAN0; Domain=protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion; Path=/; HttpOnly; Secure; Max-Age=7776000
< set-cookie: Version=default; Path=/; Secure; Max-Age=7776000
< pragma: no-cache
< content-type: text/html; charset=UTF-8
< content-security-policy: default-src 'self'; connect-src 'self' blob:; script-src 'self' blob: 'sha256-eAhF1Kdccp0BTXM6nMW7SYBdV0c3fZwzcC177TQ692g='; style-src 'self' 'unsafe-inline'; img-src http: https: data: blob: cid:; frame-src 'self' blob: https://secure.protonmail.com; object-src 'self' blob:; child-src 'self' data: blob:; report-uri https://protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/api/reports/csp;
< strict-transport-security: max-age=31536000; includeSubDomains; preload
< expect-ct: max-age=2592000, enforce, report-uri="https://protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/api/reports/tls"
< x-frame-options: deny
< x-content-type-options: nosniff
< x-xss-protection: 1; mode=block; report=https://protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/api/reports/csp
< referrer-policy: strict-origin-when-cross-origin
< x-permitted-cross-domain-policies: none
<
```

**Comment by OneWhiteBird** [comment]
> > Regarding tor support they have disabled it today as reported on their twitter: [Ctemplar Twitter](https://nitter.net/RealCTemplar/status/1458775445202157570). I have checked it manually and when logging in it redirects to ctemplar.com.
> 
> Still seems to be the case
> 
> ```
> > GET / HTTP/1.1
> > Host: mail.ctemplarpizuduxk3fkwrieizstx33kg5chlvrh37nz73pv5smsvl6ad.onion
> > User-Agent: curl/7.80.0
> > Accept: */*
> >
> * Mark bundle as not supporting multiuse
> < HTTP/1.1 301 Moved Permanently
> < Server: nginx
> < Date: Wed, 01 Dec 2021 13:15:40 GMT
> < Content-Type: text/html
> < Content-Length: 178
> < Connection: keep-alive
> < Location: https://mail.ctemplar.com/
> <
> ```
> 
> Worth noting that Protonmail still has this enabled
> 
> ```
> > GET /login HTTP/1.1
> > Host: protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion
> > User-Agent: curl/7.80.0
> > Accept: */*
> >
> * TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
> * TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
> * old SSL session ID is stale, removing
> * Mark bundle as not supporting multiuse
> < HTTP/1.1 200 OK
> < date: Wed, 01 Dec 2021 13:18:32 GMT
> < last-modified: Sun, 31 Jan 2021 09:41:46 GMT
> < accept-ranges: bytes
> < content-length: 3309
> < vary: Accept-Encoding
> < cache-control: max-age=0, no-cache, no-store, must-revalidate
> < expires: Wed, 11 Jan 1984 05:00:00 GMT
> < set-cookie: Session-Id=Yad2KC3wGVsmJJPzMw5P0QAAAN0; Domain=protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion; Path=/; HttpOnly; Secure; Max-Age=7776000
> < set-cookie: Version=default; Path=/; Secure; Max-Age=7776000
> < pragma: no-cache
> < content-type: text/html; charset=UTF-8
> < content-security-policy: default-src 'self'; connect-src 'self' blob:; script-src 'self' blob: 'sha256-eAhF1Kdccp0BTXM6nMW7SYBdV0c3fZwzcC177TQ692g='; style-src 'self' 'unsafe-inline'; img-src http: https: data: blob: cid:; frame-src 'self' blob: https://secure.protonmail.com; object-src 'self' blob:; child-src 'self' data: blob:; report-uri https://protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/api/reports/csp;
> < strict-transport-security: max-age=31536000; includeSubDomains; preload
> < expect-ct: max-age=2592000, enforce, report-uri="https://protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/api/reports/tls"
> < x-frame-options: deny
> < x-content-type-options: nosniff
> < x-xss-protection: 1; mode=block; report=https://protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/api/reports/csp
> < referrer-policy: strict-origin-when-cross-origin
> < x-permitted-cross-domain-policies: none
> <
> ```

Regarding CTemplar, it is correct. Until the [CORS](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/32865) issue is fixed, all the API requests using Tor Browser through the .onion domain will not work. So we have decided to redirect the users to our clearnet domain, until it is fixed.

**Comment by dngray** [comment]
I think this one is ready for review/merge.

**Review by freddy-m — APPROVED** [review]
LGTM!

**Pull request merged**